packages/salesforcedx-vscode-automation-tests/test/testData/sampleAnalyticsTemplateData.ts

```diff
@@ -0,0 +1,234 @@
+  '    },',
+  '    "required": true,',
+  '    "excludeSelected": false,',
+  '    "excludes": [],',
+  '    "variableType": {',
+  '      "required": [',
+  '        "createAllExternalFiles",',
+  '        "createAllDashboards",',
+  '        "createAllImages",',
+  '        "createAllDatasetFiles",',
+  '        "createAllLenses",',
+  '        "createDataflow"',
+  '      ],',
+  '      "type": "ObjectType",',
+  '      "properties": {',
+  '        "createAllDashboards": {',
+  '          "type": "BooleanType",',
+  '          "enums": [true, false]',
+  '        },',
+  '        "createAllLenses": {',
+  '          "type": "BooleanType",',
+  '          "enums": [true, false]',
+  '        },',
+  '        "createAllExternalFiles": {',
+  '          "type": "BooleanType",',
+  '          "enums": [true, false]',
+  '        },',
+  '        "createDataflow": {',
+  '          "type": "BooleanType",',
+  '          "enums": [true, false]',
+  '        },',
+  '        "createAllDatasetFiles": {',
+  '          "type": "BooleanType",',
+  '          "enums": [true, false]',
+  '        },',
+  '        "createAllImages": {',
+  '          "type": "BooleanType",',
+  '          "enums": [true, false]',
+  '        }',
+  '      },',
+  '      "strictValidation": true',
+  '    }',
+  '  }',
+  '}'
+].join('\n');
+
+export const ui = ['{', '  "pages": [],', '  "displayMessages": []', '}'].join('\n');
+
+export const templateToAppRules = ['{', '  "constants": [],', '  "macros": [],', '  "rules": []', '}'].join('\n');
+
+export const templateInfo = [
+  '{',
+  '  "templateType": "app",',
+  '  "label": "sat1",',
+  '  "name": "sat1",',
+  '  "description": "",',
+  '  "assetVersion": 57.0,',
+  '  "variableDefinition": "variables.json",',
+  '  "uiDefinition": "ui.json",',
+  '  "rules": [',
+  '    {',
+  '      "type": "templateToApp",',
+  '      "file": "template-to-app-rules.json"',
+  '    },',
+  '    {',
+  '      "type": "appToTemplate",',
+  '      "file": "app-to-template-rules.json"',
+  '    }',
+  '  ],',
+  '  "releaseInfo": {',
+  '    "templateVersion": "1.0",',
+  '    "notesFile": "releaseNotes.html"',
+  '  },',
+  '  "folderDefinition": "folder.json",',
+  '  "externalFiles": [],',
+  '  "lenses": [],',
+  '  "dashboards": [',
+  '    {',
+  '      "label": "sat1Dashboard",',
+  '      "name": "sat1Dashboard_tp",',
+  '      "condition": "${Variables.Overrides.createAllDashboards}",',
+  '      "file": "dashboards/sat1Dashboard.json"',
+  '    }',
+  '  ],',
+  '  "eltDataflows": [],',
+  '  "datasetFiles": [],',
+  '  "imageFiles": [],',
+  '  "extendedTypes": {},',
+  '  "templateDependencies": [],',
+  '  "icons": {',
+  '    "appBadge": {',
+  '      "name": "16.png"',
+  '    },',
+  '    "templateBadge": {',
+  '      "name": "default.png"',
+  '    },',
+  '    "templatePreviews": []',
+  '  }',
+  '}'
+].join('\n');
+
+export const releaseNotes = '<h2>Release Notes for Template</h2>';
+
+export const folder = [
```

Comment:
you can `import` a json file in TS.  we don't have to build them all out of strings.